data/community.yml

```diff
@@ -1,140 +1,41 @@
-          
-    # team member
-    - image : "images/people/samfull.webp"
-      imageAlt : "images/people/samfull.jpg"
-      name : "Sam Ellis"
-      designation : "Software Scientist for MolSSI"
-      content : "Primary Requirements Engineer for the QCArchive project, keeps the rest of us from the dreaded scope creep"
-
-          
-    # team member
-    - image : "images/people/chayafull.webp"
-      imageAlt : "images/people/chayafull.jpg"
-      name : "Chaya Stern"
-      designation : "Open Forcefield Power User"
-      content : "Scientist on behalf of Open Forcefield who has been using QCArchive since the beginning and helping shape end-user interface and development"
+    # Open Force field Consortium
+    - image : "images/collaborators/OpenFFVector.png"
+#      imageAlt : "images/collaborators/OpenFFVector.png"
+      name : "Open Force Field Consortium"
+      designation : "Collaborator and Sponsor"
+      content : "The <a href='url'>OpenFF Consortium</a> is an academic-industry collaboration designed to improve computer models used to
+                 guide pharmaceutical drug discovery. They are a major sponsor of the QCArchive and the customer for the use case of
+                 generating data for the parameters of a bespoke force field. They fragment and provide inputs for
+                 torsion scans, then provide those inputs to the QCArchive ecosystem for computation. Those computations
+                 then help generate parameters for the force fields based on atomic elements and connectivity rather
+                 than atom typing."
+#      socialIcon :
+#        - icon : "tf-ion-social-github"
+#          URL : "https://github.com/openforcefield/openforcefield"
+
+    # Distributed Driver
+    # From Lori: seamless embarrassingly parallel capabilities, deconvolute driver code, separate generic QC procedures driver from QC methods backend, checkpointing
+    - image: "images/collaborators/distributedDriver.png"
+      name: "Distributed Driver"
+      designation: "Developer and Extension"
+      content: "The Distributed Driver is a Psi4 compute driver designed to handle nested, conditional, and almost
+                programmatic operations in an iterative fashion. Multiple calculation managers and all their children
+                child managers can be staged together such that compute commands issued at parents are propagated to the
+                children.
+                This sort of distribution and job staging is exactly what the Fractal job queue and distributed
+                managers were designed to handle. Engine's schema-based input enables the type of programmatic execution
+                needed to handle these computations. Fractal, Engine, and Schema together vastly reduce the barrier to
+                efficiently setup and implement this technology and the Distributed Driver handles the top level task
+                creation logic and post-compute processing."
```

Comment:
Not wholly sure what the first paragraph means, but I'll believe you. the "children child managers" needs rewording, though.